core/src/main/java/feign/Response.java

```diff
@@ -29,6 +29,7 @@
   private final Map<String, Collection<String>> headers;
   private final Body body;
   private final Request request;
+  private final String protocolVersion;
```

Comment:
Wondering if want to make this an enum or we wan't to leave it open so people can have `FTPES` or any other protocols they want to

Reply:
I think that we can use enum: there are limited list of protocol version - HTTP/1.0, HTTP/1.1, HTTP/2.0 (HTTP/2). In the future we can add new values if they are needed.

I have added `Request.ProtocolVersion` enum and `Util.enumForName` method that helps to find enum value by any of strings: _HTTP_1_0_, _HTTP/1.0_, _HTTP_2_, _HTTP/2.0_ etc.